networking_cisco/plugins/cisco/db/l3/l3_router_appliance_db.py

```diff
@@ -453,6 +450,16 @@ def add_router_interface(self, context, router_id, interface_info):
         routers = [self.get_router(context, router_id)]
         self.add_type_and_hosting_device_info(context.elevated(), routers[0])
         if driver:
+            if by_subnet:
```

Comment:
not sure why we have to get the port_db here again.....it looks we already got it while calling the precommit()? I'm not following this change so....